docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -5,7 +5,204 @@ sidebar_position: 1
 # Creating Service Binding
 
 In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
+- Being part of the "known" services
 - Exposing binding-metadatas by default
 If it's not the case, please refer to the following section(#)
 
+
+### Service Binding Resource
+
+Service Binding resources use `binding.operators.coreos.com` group with  `v1alpha1` version and `ServiceBinding` kind.
+
+A sample Service Binding resource that connects a nodejs application to a postgresql database:
+
+
+```yaml
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+
+metadata:
+  name: binding-request 
+
+spec:
+
+  application:
+    name: nodejs-rest-http-crud
+    group: apps
+    version: v1
+    resource: deployments
+
+  services:
+  - group: postgresql.dev
+    version: v1alpha1
+    kind: Database
+    name: db-demo
+```
+
+The `spec` of a Service Binding resource has two sections:
+- application
+- services
+
+### Application
+
+An application is a process running within a container. Examples include a NodeJS Express application, a Ruby Rails application or a Spring Boot application.
+
+Create a NodeJS application:
+
+```yaml
+kubectl apply -f - << EOD
+---
+apiVersion: apps/v1 ############################### GROUP/VERSION
+kind: Deployment    ############################### RESOURCE
+
+metadata:
+  name: nodejs-rest-http-crud  #################### NAME
+  labels:
+    app: nodejs-rest-http-crud
+    runtime: nodejs
+    runtime-version: 14-ubi7
+
+spec:
+  selector:
+    matchLabels:
+      app: nodejs-rest-http-crud
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nodejs-rest-http-crud
+        deploymentconfig: nodejs-rest-http-crud
+    spec:
+      containers:
+      - name: nodejs-rest-http-crud
+        image: nodejs-rest-http-crud:latest
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+EOD
+```
+
+Application can be a podSpec based resource like "Deployment" or "DeploymentConfig" or a non podSepc based resource like a Custom Resource Definition.
+
+### Service
+
+Apply the following CatalogSource:
```

Comment:
Similar comment from Application section.
We should explain what the service in the ServiceBinding CR is. each fields, and a description + explanation of the format expected.

Reply:
Addressed